### PR TITLE
Back bug fix

### PR DESCRIPTION
**Pull request by agonxalez**

### CODE REVIEW

File: components/user-profile/index.tsx [hunk 1]
```diff
@@ -19,13 +19,18 @@ const UserProfile = (id: any) => {
   const profileData = data as unknown as FullProfile;
   const [tabIndex, setTabIndex] = useState(0);
   const prevPage = document?.referrer.toString()
+  console.log(prevPage)
```
Comment (bdurel) [inline components/user-profile/index.tsx:22]: remove console.log

**Review by bdurel — CHANGES_REQUESTED** [review]
It works on when landing on page from outside link but does not work when new tab is opened. I think it should navigate back to homepage in both cases.

File: components/user-profile/index.tsx (hunk 1 above)
Comment (blamboy090) [reply components/user-profile/index.tsx:22]: I second this

**Comment by blamboy090** [comment]
Correct functionality has been observed during QA of this fix, with the exception of logging out and using a copied profile link in the browser, while I do not believe this pertains to the AC of this card, I think it is addressed in a different card 
First screen shot is for when you refresh a duplicated tab

Second is when you copy user url when logged out

**Pull request merged**